src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/workingLists.types.js

```diff
@@ -150,4 +151,10 @@ export type InterfaceProps<InputDataSource> = $ReadOnly<{|
     stickyFilters?: StickyFilters,
     templates?: WorkingListTemplates,
     templatesLoading: boolean,
+    updating: boolean,
+    updatingWithDialog: boolean,
+|}>;
+
+export type WorkingListsOutputProps = $ReadOnly<{|
+    ...InterfaceProps,
```

Comment:
Thinking if this is really necessary. Perhaps we can reuse the `InterfaceProps` and give it another name like you do here with the  `WorkingListsOutputProps`?

Reply:
Not entirely sure what you meant here

Reply:
changed to `export type WorkingListsOutputProps = InterfaceProps;` (this is maybe what you meant)